docs/platform-and-architecture/roles-in-marlowe.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Roles in Marlowe
+sidebar_position: 5
+---
+
+Generally speaking, participants are actors in a smart contract. These participants fulfill roles that specify the ability to execute certain actions in the contract.
+
+Roles can be hardcoded addresses, pre-determined before launching a contract or determined after launching a contract.
+
+These roles are uniquely identified using role tokens.
+
+## Role tokens
+
+Role tokens are used to identify roles to authorize transactions providing additional security and flexibility for your smart contracts. They are handled implicitly by Marlowe Runtime. 
+
+Each participant in a Marlowe contract has a role and each role has its own token. These tokens can be transferred between users in a peer-to-peer manner, allowing a participant to give their role to someone else.
+
+There could be one or many tokens for a specific role.
+
+Role tokens are distributed at contract initialization.
+
+You can view these role tokens in your wallet under NFTs/Assets with a name reflective of the name specified for the role in the smart contract. (In Nami, Lace does not carry the name over from the DApp to the asset.) 
```

Comment:
@nstanford5 I'd like to clarify this statement about Nami and Lace. I'm not quite clear what you mean by "In Nami, Lace does not carry the name over from the DApp to the asset." Could you share a little bit more detail about it?

Reply:
Sure -- when you give your Role a name in Marlowe Playground or TS-SDK, such as "SavingsAccount" -- using Nami wallet, it will carry over that name and display it with the Token under Assets.

If you are using Lace, the Role Token shows up as an NFT with a name that is not relevant to your Smart Contract. Something like "SingleNFTPreprod" -- even though it is a Role Token. Likely a feature that the Lace team should consider implementing.

For context of this page, I would like them to know where to find these assets -- but they are different in each wallet, so what is there now is probably not the best.

Maybe -- "You can view these Role Tokens in your wallet as Assets or NFTs"